hardware/dram_rtl_sim/Bender.yml

```diff
@@ -0,0 +1,22 @@
+# Copyright 2021 ETH Zurich and University of Bologna.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+package:
```

Comment:
It looks like `dram_rtl_sim` is a full dependency. I think it would make more sense to create a dedicated repository for it that can be included as a dependency instead of flattening it in MemPool. Especially since it's almost 200 files and more than 2 million lines of code. :astonished: Is it also used in other projects or just here?